File: SignUpUi/sign-up-ui/src/app/core/user.model.ts
export class UserModel {
    id:string;
    email: string;
    firstName: string;
    lastName: string;
    address: string;
    job: string;
    birthDate: Date;

    constructor(id:string, email: string, firstName: string, lastName: string, address: string, job: string, birthDate:Date) {
      this.id = id;
      this.email = email;
      this.firstName = firstName;
      this.lastName = lastName;
      this.address = address;
      this.job = job;
      this.birthDate = birthDate;
    }
  }

File: IdentityServer/Common/Models/UserModel.cs
using Amazon.DynamoDBv2.DataModel;
using Common.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.Models
{
    [DynamoDBTable("Users")]
    public class UserModel
    {
        [DynamoDBHashKey]
        public string Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Job { get; set; }
        public string RegistrationType { get; set; }
        public string RegistrationKey { get; set; }
        public DateTime BirthDate { get; set; }
    }
}
